libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -207,10 +243,20 @@ AP_GPS_SBF::parse(uint8_t temp)
 
                 // received the result, lets assess it
                 if (sbf_msg.data.bytes[0] == ':') {
+                    const char* initialization_blob[5];
```

Comment:
This is overly complex, all you actually need is a single pointer which you put the address of the appropriate blob into, rather then memcpying the array, which particularly doesn't work as soon as any of the entries have more or fewer entries in them. (You'd need a set of static_asserts() to help catch this in the future)